docs/DeveloperGuide.md

```diff
@@ -359,11 +452,15 @@ to identify the indices present in ` 3`. `MarkingCommandParser#parse()` then ini
 Step 3. `MarkCommand#execute()` is then called, which will in turn call `Model#checkForUnmarkedPerson()` on the applicants
```

Comment:
Even though this is not part of your PR, I noticed an error in step 3 where it should be `UnmarkCommand#execute()`. You could change it in this PR?